test/case/ietf_interfaces/tunnel_ttl/test.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env python3
+"""GRE Tunnel TTL Verification
```

Comment:
This is a GRE only test, why not name it gre_ttl? The other tests are named tunnel_ because they are share between gre and vxlan.